packages/block-library/src/template-part/index.php

```diff
@@ -63,7 +63,8 @@ function render_block_core_template_part( $attributes ) {
 	}
 	$content = do_shortcode( $content );
 
-	return '<div>' . str_replace( ']]>', ']]&gt;', $content ) . '</div>';
+	$html_tag = $attributes['tagName'];
```

Comment:
is there some kind of escaping function we need to run here?

Reply:
I think `esc_attr` may still be useful in this case?

Reply:
Yeah. I guess, if it _does_ end up escaping anything, it will be an invalid HTML tag, but there would be bigger issues in that case anyways :P